src/scenes/WireInstructions.js

```diff
@@ -35,9 +41,20 @@ class WireInstructionsComponent extends Component<Props, State> {
         {arg}
       </div>
       }
-      return <div className={classes.individualLinesText}>
-      {arg}
-    </div>
+      if(arg === '') {
+        return <div className={classes.subHeaderText}>
+      </div>
+      }
+      return <div className={classes.individualLinesText} onClick={() => {
+          navigator.clipboard.writeText(arg)
```

Comment:
If this works, do we need the other copyToClipboard methods as props on the other components? If it doesn't, does it need to be fixed.